client/state/selectors/test/is-requesting-timezones.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingTimezones } from '../';
```

Comment:
We can go ahead and delete this file, since we're using the data/http layer and don't store this state explicitly